pkg/logql/engine.go

```diff
@@ -226,7 +216,14 @@ func (q *query) Exec(ctx context.Context) (logqlmodel.Result, error) {
 }
 
 func (q *query) Eval(ctx context.Context) (promql_parser.Value, error) {
-	ctx, cancel := context.WithTimeout(ctx, q.timeout)
+	queryTimeout := time.Minute * 2
```

Comment:
We're changing the default timeout to 2m here?
This should definitely be documented.

Reply:
The timeout is only 2 minutes for when we can't fetch the tenant. But regardless of it, no reason to change this to 2 minutes so I'm returning it to 5 minutes.